tests/test_cli.py

```diff
@@ -312,7 +312,8 @@ def test_datasets_import(data_file, data_repository, runner, project, client):
     # create a dataset
     result = runner.invoke(cli.cli, ['dataset', 'create', 'dataset'])
     assert result.exit_code == 0
-    assert (client.renku_datasets_path / 'dataset' / client.METADATA).exists()
+    with client.with_dataset('dataset') as dataset:
```

Comment:
Are we running pytest in parallel on some environments?

Reply:
You can with `pytest -n=auto`.

Reply:
Thanks, I know. :+1: 
I'm just asking generally for environments since I'm too lazy to check .travis.yml. 

Nevermind, I answered myself in meantime. :+1: :smile: